CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ Version template:
  -->
  
 # Alfred Telemetry Changelog
+## [0.10.0] - 2023-08-31
+### Added
+* Added support for alfresco 7.4 [#142]
```

Comment:
Is the # correct?
`[something]` is normally just a hyperlink, but you still need to define it, e.g.: 
```
[123]
[#123]: https://github.com/xenit-eu/alfred-telemetry/pull/123
```

See further below in the changelog